packages/frontend/mobile/src/views/all-docs/style.css.ts

```diff
@@ -1,20 +1,32 @@
 import { cssVarV2 } from '@toeverything/theme/v2';
 import { style } from '@vanilla-extract/css';
 
-export const header = style({
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'space-between',
-  gap: 16,
-  padding: '16px 16px 0px 16px',
+const headerContentHeight = 56;
+const headerPaddingTop = 16;
```

Comment:
use `env(safe-area-inset-top)` when available for padding top?

Reply:
will be handled with `pwa`